team.md

```diff
@@ -4,10 +4,23 @@ title: Team
 permalink: /team/
 ---
 
-# Josh Kuun
+# Robin Hood Army (RHA) Web Development
 
-# Kausar Khizra
+A zero fund volunteer organization that collects extra food from restaurants and communities to serve the ones in need.
 
-# Tylar Farmer
+# Goals
+- Improve the Search Engine Optimization (SEO)
+- Improve overall usability of the website, add features
 
-# Vidya Ananda
\ No newline at end of file
+
+# Team Spring 2024
+## Josh Kuun
+
+## Kausar Khizra
+
+## Tylar Farmer
+
+## Vidya Ananda
+
+# Lighthouse Score of Homepage
+![Alt text](lighthouse-report.png)
```

Comment:
I think the Lighthouse scores need to be text instead of an image. From the P3 Team Web Page description:
> Text listing of Lighthouse scores for your home page only

Reply:
Yes, I put it just as placeholder for now, screenshot does not even cover the complete report which is quite lengthy.  In text format, we've two options: json and html? Should we go for iframe to embed it in our page?